crates/router/src/compatibility/stripe/payment_intents/types.rs

```diff
@@ -143,12 +164,14 @@ pub struct StripePaymentIntentRequest {
     pub shipping: Option<Shipping>,
     pub statement_descriptor: Option<String>,
     pub statement_descriptor_suffix: Option<String>,
-    pub metadata: Option<api_models::payments::Metadata>,
+    #[schema(value_type = Option<Object>, example = r#"{ "udf1": "some-value", "udf2": "some-value" }"#)]
```

Comment:
You can remove `schema` . we can add `schema` for whole stripe compatibility in a separate PR if required.

Reply:
done